machine/v1alpha1/types_openstack.go

```diff
@@ -92,6 +92,12 @@ type OpenstackProviderSpec struct {
 	// The volume metadata to boot from
 	RootVolume *RootVolume `json:"rootVolume,omitempty"`
 
+	// AdditionalBlockDevices is a list of specifications for additional block devices to attach to the server instance
+	// +optional
+	// +listType=map
```

Comment:
are the list* parameters useful?

Reply:
You need a map key, which you have, so I think this is correct. I think it's best practice to include even though there's no effect as this is raw extension